src/components/Dog.jsx

```diff
@@ -1,31 +1,22 @@
-import React, { useState } from "react";
-import PropTypes from "prop-types";
-import "./Dog.css";
+import PropTypes from 'prop-types';
+import './Dog.css';
 
 const Dog = (props) => {
-    const [chipNum, setChipNum] = useState(props.chip);
-
-    //https://www.w3schools.com/js/js_random.asp
-    const getRndInteger = (min, max) => {
-      return Math.floor(Math.random() * (max - min)) + min;
-    };
-  
-    const createChip = () => {
-      setChipNum(getRndInteger(1000, 9999));
-      console.log();
-    };
-    
     return (
         <li>
           <h2>
             {props.name}
-            <button className="delete__dog">delete</button>
+            <button onClick={() => props.deleteDogCallback(props.id)}>
```

Comment:
The fact that we _cannot_ use the passed in functions directly here really makes me want to be sure that we highlight why we were able to in the previous branch.